.vscode/settings.template.json

```diff
@@ -47,8 +47,6 @@
     "./services/web/server/src",
     "./services/web/server/tests/unit/with_dbs"
   ],
-  "python.linting.pylintEnabled": true,
-  "python.linting.enabled": true,
```

Comment:
why are these removed?

Reply:
they are deprecated. that is a warning I get now everytime I start vscode

Reply:
I think they changed how python linters are integrated: https://code.visualstudio.com/docs/python/linting